the_iron_yelp/restaurants_server.rb

```diff
@@ -0,0 +1,59 @@
+require "webrick"
+require "erb"
+require "json"
+require "./restaurants1"
+
+server = WEBrick::HTTPServer.new(:Port => 8000)
+
+restaurants_file = File.read "restaurants.json"
+restaurants_array = JSON.parse restaurants_file 
+
+  server.mount_proc "/home" do |request, response|
+	response.body = File.read "yelp_home.html"
+  end
+
+  server.mount_proc "/shutdown" do |request, response|
+	response.body = "Bye Bye"
+	server.shutdown
+  end
+  
+  server.mount_proc "/The Rusty Spoon" do | request, response|
+   @restaurants = Restaurant.named "The Rusty Spoon"
```

Comment:
that's a bit confusing, since @restaurants looks like it's going to be plural, but there's only one Rusty Spoon.